.travis.yml

```diff
@@ -1,32 +1,53 @@
 language: objective-c
 before_install:
   - brew update
-  - brew reinstall xctool
 install:
-  - brew install watchman
-  - npm install
-before_script:
-  - npm test
-  - (npm start > packager.log 2>&1 &) && echo $! > packager.pid
-
-after_script:
-  - pkill -9 -F packager.pid
-  - cat packager.log
-  - rm packager.log
-  - rm packager.pid
-
-# Automatically publish the website
-after_success:
+  - brew reinstall xctool watchman
+
+script:
 - |
-  if [ "$TRAVIS_PULL_REQUEST" = false ] && [ "$TRAVIS_BRANCH" = master ]; then
-    echo "machine github.com login reactjs-bot password $GITHUB_TOKEN" >~/.netrc
-    cd website
-    npm install
-    ./setup.sh
-    ./publish.sh
+  if [ "$TEST_TYPE" = objc ]
+  then
+
+    (npm start > packager.log 2>&1 &)
+    echo $! > packager.pid
+
+    xctool \
+      -project Examples/UIExplorer/UIExplorer.xcodeproj \
+      -scheme UIExplorer -sdk iphonesimulator8.1
+      test
+
+    pkill -9 -F packager.pid
+    cat packager.log
+    rm packager.log packager.pid
+
+  elif [ "$TEST_TYPE" = js ]
+  then
+
+    npm test
+
+  elif [ "$TEST_TYPE" = build_website ]
+  then
+
+    # Automatically publish the website
+    if [ "$TRAVIS_PULL_REQUEST" = false ] && [ "$TRAVIS_BRANCH" = master ]; then
```

Comment:
It's not?

![image](https://cloud.githubusercontent.com/assets/6820/6955982/44bd59d6-d89b-11e4-9303-f6d6b1ca7e23.png)